test/spec/modules/smartyadsBidAdapter_spec.js

```diff
@@ -57,6 +59,19 @@ describe('SmartyadsAdapter', function () {
       expect(data.placements).to.be.an('array').that.is.empty;
     });
   });
+
+  describe('with COPPA', function() {
+    it('should send the Coppa "required" flag set to "1" in the request', function () {
+      sinon.stub(config, 'getConfig')
+        .withArgs('coppa')
+        .returns(true);
```

Comment:
It is a better idea to set your stubs in beforeEach and reset them in afterEach.

If for some reason your test fails before the `config.getConfig.restore();` then the next adapter tests which run will have this stubbed still which can cause flakey failures.